src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/SecurityController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SecurityController extends AbstractController
+{
+    /**
+    * @Route("/logout", name="app_logout", methods={"GET"})
```

Comment:
If you keep this construction in favour of my suggestion then please:

1. Move this route annotation to the routes.yml
2. Remove the app_ prefix from the route name
3. +1 for adding the comment, explaining why this was added. But in my opinion this indicates a smelly solution.

Reply:
Removed this as per your suggestion above.